google/docs/tool.gpt

```diff
@@ -1,18 +1,38 @@
 ---
-Name: Google Docs 
+Name: Google Docs
 Metadata: bundle: true
 Description: Tools for managing Google Docs
-Share Tools: Read Doc
+Share Tools: Read Google Doc, Update Google Doc
 
 ---
-Name: Read Doc
-Description: Returns the content of a Google Doc in markdown format
+Name: Read Google Doc
+Description: Returns the content of a Google Doc in markdown format.
 Share Context: Google Docs Context
 Credential: ../credential
 Param: doc_ref: Google Docs ID or share link of the document to read
 
 #!/usr/bin/env python3 ${GPTSCRIPT_TOOL_DIR}/read_doc.py
 
+---
+Name: Update Google Doc
+Description: Updates a Google Doc
+Share Tools: Create Google Doc
+Share Context: Google Docs Context, Update Google Doc Context
+Credential: ../credential
+Param: doc_ref: Google Docs ID or share link of the document to read
+Param: new_doc_content: Markdown formatted content to replace the existing content of the document with
+
+#!/usr/bin/env python3 ${GPTSCRIPT_TOOL_DIR}/update_doc.py
+
+---
+Name: Create Google Doc
+Description: Creates a blank Google Doc
+Share Context: Google Docs Context
+Credential: ../credential
+Param: doc_title: The title of the document to create
```

Comment:
Any reason to not allow creating a document with content?

Reply:
They're two separate operations in the API. To me it seemed like dividing the responsibilities the same way results in simpler tools.

Reply:
Interesting. Thanks for the explanation!